packages/calcite-components/src/components/meter/meter.tsx

```diff
@@ -0,0 +1,291 @@
+import { Component, Element, Fragment, h, Host, Prop, State, VNode, Watch } from "@stencil/core";
+import { Appearance, Kind, Scale } from "../interfaces";
+import { CSS } from "./resources";
+
+import {
+  connectLocalized,
+  disconnectLocalized,
+  LocalizedComponent,
+  NumberingSystem,
+  numberStringFormatter,
+} from "../../utils/locale";
+
+@Component({
+  tag: "calcite-meter",
+  styleUrl: "meter.scss",
+  shadow: true,
+})
+export class Meter implements LocalizedComponent {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+  @Element() el: HTMLCalciteMeterElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  //--------------------------------------------------------------------------
+
+  /**
+   * When `true`, interaction is prevented and the component is displayed with lower opacity.
+   */
+  @Prop({ reflect: true }) disabled = false;
+
+  /**  Accessible name for the component. */
+  @Prop() label!: string;
+
+  /** Specifies the size of the component. */
+  @Prop({ reflect: true }) scale: Scale = "m";
+
+  /** Specifies the current value of the component. */
+  @Prop() value: number;
+
+  /** Specifies the miniumum possible value of the component. */
+  @Prop() min = 0;
+
+  /** Specifies the maximum possible value of the component. */
+  @Prop() max = 100;
+
+  /** Optionally provide a low value - above this, when `fillType` is 'range', the component display the appropriate color  */
+  @Prop() low: number;
+
+  /** Optionally provide a high value - above this, when `fillType` is 'range', the component display the appropriate color  */
+  @Prop() high: number;
+
+  /** Does the fill appear as a single "brand" color or does it display indications of low / high / "success", "danger", or "warning" based on low, high if provided, or value compared to max */
+  @Prop() fillType: "single" | "range" = "range";
+
+  /** Specifies the Unicode numeral system used by the component for localization. */
+  @Prop() numberingSystem: NumberingSystem;
+
+  /** When `true`, number values are displayed with a group separator corresponding to the language and country format. */
+  @Prop({ reflect: true }) groupSeparator = false;
+
+  /** The type of appearance of the meter */
+  @Prop() appearanceType: Extract<"outline" | "outline-fill" | "solid", Appearance> =
+    "outline-fill";
+
+  /** Determine if the meter should be displayed with a percent based fill, or based on discreet numbers. This will affect displayed labels */
```

Comment:
WDYT about the following?:

```Specifies the component's unit type. When `displayValue` or `displayRangeLabels` is `true`, also specifies the component's labels.```

Reply:
Maybe `Specifies the format of displayed labels, when either "valueLabel" and/or "rangeLabels" are true.`

Reply:
Could we flip your recommendation, for alignment with other docs where we specify the props conditions first? Something like:

```When either `valueLabel` and/or `rangeLabels` are `true`, specifies the format of displayed labels.```